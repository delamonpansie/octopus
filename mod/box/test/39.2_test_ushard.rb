#!/usr/bin/ruby
# coding: utf-8

$: << File.dirname($0)
require '39_test_ushard'

# non existent shard
log_try { $one.select 1, :shard => 33 }

