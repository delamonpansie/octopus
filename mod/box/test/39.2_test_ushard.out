# one.ping()
:pong

# two.ping()
:pong

# one.select(1, {:shard=>33})
Failed with: {code: 0x102, message: 'no such shard'}
